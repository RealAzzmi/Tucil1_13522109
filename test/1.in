6
11 11
E9 E9 E9 7A BD 1C E9 BD 55 7A 1C 
1C 7A 7A BD 55 BD 1C E9 55 E9 BD 
E9 7A 7A 55 55 E9 BD BD 7A 55 1C 
1C 1C BD 55 7A BD E9 7A BD 7A BD 
7A 7A E9 55 55 1C 1C 7A 55 55 7A 
55 1C 55 55 1C BD E9 55 55 55 E9 
BD BD 1C 55 7A 1C 1C 55 E9 BD 1C 
BD 1C BD 7A 1C E9 1C BD E9 BD 1C 
55 55 7A BD 1C BD E9 E9 1C BD BD 
1C BD BD E9 55 55 E9 BD 1C 55 E9 
7A 7A 1C 7A BD 55 E9 7A E9 BD 7A 
1000
7A 7A 
3
1C E9 
-7
55 55 
10
BD E9 
-13
1C 55 
6
E9 55 
1
55 1C 
10
1C 55 
-7
E9 1C 
7
E9 BD 
-3
55 BD 
2
1C 1C 
0
E9 7A 
-10
7A 55 
-11
E9 1C 
-10
E9 55 
10
E9 55 
-8
55 7A 
9
55 7A 
-13
BD 1C 
10
E9 1C 
4
BD BD 
-7
BD BD 
-7
1C 55 
3
7A 55 
7
55 55 
-7
1C 7A 
-12
7A BD 
3
E9 E9 
3
E9 1C 
4
E9 1C 
-5
7A 1C 
7
1C 55 
9
55 1C 
5
55 E9 
6
7A E9 
-6
55 1C 
-10
BD 1C 
-6
BD 55 
7
1C E9 
8
55 1C 
-13
55 55 
-4
E9 1C 
-1
1C 1C 
1
7A 1C 
-1
7A 7A 
-11
BD BD 
2
7A 7A 
7
55 55 
-5
BD E9 
3
1C E9 
-10
7A 1C 
10
BD 1C 
4
7A E9 
-5
7A 1C 
-5
BD 55 
1
BD 7A 
10
7A 7A 
7
BD E9 
-7
55 BD 
7
BD 7A 
-10
1C E9 
-4
E9 BD 
6
7A 1C 
-12
BD 55 
-5
55 BD 
-1
7A 55 
-9
55 7A 
7
55 1C 
6
E9 E9 
5
E9 1C 
6
7A E9 
-11
1C BD 
-1
1C E9 
7
55 E9 
4
55 BD 
-7
55 55 
-10
1C BD 
5
1C 7A 
1
55 1C 
1
55 7A 
-3
BD 1C 
5
1C 55 
-6
BD E9 
1
7A 7A 
-10
1C 55 
-2
55 1C 
-1
E9 7A 
-9
7A 7A 
-3
55 BD 
-9
BD 1C 
-3
1C 7A 
-4
E9 E9 
-11
1C 1C 
-3
E9 55 
0
1C E9 
-13
BD 1C 
10
1C BD 
-8
7A 1C 
-11
1C E9 
-2
E9 1C 
-3
E9 BD 
-7
7A 7A 
3
7A 1C 
-2
BD 1C 
4
E9 1C 
7
BD 1C 
9
1C BD 
5
55 BD 
-2
7A BD 
-6
7A 1C 
4
55 7A 
-1
1C 55 
-2
E9 E9 
-7
7A E9 
-11
BD 55 
-7
55 1C 
9
55 1C 
3
55 E9 
7
7A 7A 
-8
BD BD 
-1
55 55 
0
55 BD 
-3
BD E9 
-2
E9 E9 
10
7A E9 
-12
55 E9 
-3
7A BD 
-3
E9 1C 
1
BD 55 
6
1C 1C 
-4
E9 E9 
1
1C 1C 
4
BD 7A 
3
7A BD 
-5
7A 7A 
-8
E9 E9 
2
7A 55 
-10
E9 7A 
-1
1C 7A 
5
E9 BD 
10
BD BD 
-5
E9 1C 
-9
BD E9 
-5
7A 55 
-5
E9 7A 
8
55 E9 
4
1C 1C 
3
7A 55 
-4
BD 55 
-12
1C 1C 
-11
E9 1C 
-10
E9 BD 
-2
E9 7A 
-11
E9 55 
5
E9 BD 
0
55 1C 
1
1C BD 
5
7A BD 
-4
BD 1C 
6
E9 E9 
-9
7A 7A 
-7
55 55 
-8
BD 55 
-12
E9 55 
1
BD 55 
5
BD 7A 
7
E9 E9 
-8
1C 7A 
10
7A 7A 
-8
BD 55 
4
55 7A 
6
7A E9 
2
1C E9 
10
55 55 
0
E9 55 
4
7A BD 
5
55 BD 
-6
BD BD 
5
7A 1C 
2
BD 7A 
-9
BD E9 
2
1C BD 
3
E9 BD 
10
E9 55 
-9
7A BD 
-12
BD 1C 
-11
7A E9 
-12
55 1C 
2
1C 7A 
4
1C 7A 
-1
55 55 
0
55 7A 
-6
1C 7A 
4
BD BD 
-1
E9 55 
-12
55 55 
-1
55 55 
-3
E9 7A 
9
E9 55 
-3
7A 7A 
-8
55 E9 
8
55 BD 
1
E9 BD 
9
55 55 
-8
E9 55 
4
55 55 
-13
BD BD 
0
E9 7A 
-9
1C E9 
-7
55 55 
-8
BD BD 
-3
7A BD 
-8
E9 E9 
10
7A BD 
-11
55 55 
9
BD BD 
-7
BD 55 
-2
55 BD 
-1
E9 E9 
-1
BD E9 
-2
7A 55 
8
55 7A 
-13
55 BD 
10
1C 1C 
5
1C 55 
8
7A E9 
-9
7A 7A 
7
1C 7A 
2
1C BD 
0
7A 55 
-1
1C 7A 
-3
55 BD 
-4
7A E9 
-5
E9 1C 
-1
1C 55 
-13
1C BD 
1
BD E9 
6
BD E9 
8
BD 55 
3
55 BD 
-13
55 E9 
10
1C BD 
-7
55 7A 
8
E9 55 
-3
E9 BD 
-7
1C BD 
8
7A 7A 
10
BD 7A 
-7
55 1C 
-6
1C 7A 
3
1C 55 
1
55 BD 
-5
7A BD 
-3
E9 1C 
5
E9 7A 
9
1C 7A 
-11
55 E9 
-12
E9 BD 
-5
7A E9 
5
E9 1C 
10
E9 7A 
7
E9 55 
3
7A 7A 
9
7A 1C 
0
BD E9 
3
55 1C 
-2
1C 1C 
5
E9 7A 
-6
BD BD 
4
BD 7A 
9
BD 7A 
9
E9 E9 
-9
E9 1C 
2
1C 7A 
-1
BD 1C 
2
BD 55 
8
7A 1C 
-13
55 BD 
1
55 7A 
8
BD 1C 
-11
BD E9 
-8
E9 7A 
6
E9 55 
-7
BD BD 
-8
1C BD 
5
BD BD 
6
7A E9 
-10
E9 E9 
-12
E9 55 
-10
BD 7A 
10
E9 55 
-11
1C E9 
9
55 BD 
9
BD 7A 
4
E9 7A 
-8
1C BD 
-6
BD BD 
-7
55 55 
-1
E9 55 
-4
7A E9 
-8
55 55 
9
BD 55 
0
BD 1C 
9
E9 55 
-13
BD 7A 
10
1C 55 
-6
1C E9 
2
1C 1C 
-3
BD 55 
5
55 BD 
8
BD BD 
10
55 BD 
-13
7A E9 
0
1C BD 
8
E9 55 
8
7A 1C 
-1
55 E9 
10
7A E9 
-1
1C 1C 
-11
E9 7A 
6
1C 55 
-10
BD 55 
-6
BD 7A 
-13
1C E9 
6
E9 E9 
-12
BD 7A 
-7
E9 1C 
4
BD 55 
-2
E9 55 
-8
BD 55 
-6
1C BD 
9
E9 BD 
3
7A 55 
-2
7A 55 
-11
7A BD 
-6
7A 7A 
-3
7A BD 
-3
1C E9 
7
BD 55 
-2
7A E9 
-10
7A 7A 
-10
BD 7A 
9
55 1C 
-10
E9 7A 
-2
7A 55 
-2
BD E9 
-8
E9 1C 
7
7A 55 
-5
1C E9 
2
1C BD 
-2
BD BD 
6
1C 55 
3
1C 1C 
-6
E9 55 
-9
E9 7A 
-3
E9 7A 
1
55 E9 
0
55 7A 
-3
7A 1C 
3
7A E9 
10
E9 7A 
-4
E9 1C 
8
1C 1C 
-3
7A BD 
0
7A 1C 
8
55 E9 
-10
1C 1C 
-11
1C 7A 
-12
BD 1C 
-9
55 E9 
-2
E9 7A 
-11
E9 BD 
-5
E9 7A 
-5
1C BD 
7
E9 1C 
3
55 55 
-6
7A 1C 
-5
BD 1C 
6
1C 55 
6
1C 7A 
-3
7A BD 
4
55 55 
-6
BD 7A 
-1
55 7A 
10
E9 E9 
-1
E9 1C 
-7
E9 7A 
-2
E9 E9 
-6
55 7A 
-2
1C 55 
2
55 1C 
-1
1C BD 
9
1C 55 
-12
55 BD 
6
E9 1C 
2
BD BD 
-10
E9 BD 
-5
E9 E9 
-8
55 7A 
-12
BD 55 
2
E9 7A 
-1
55 BD 
-7
55 7A 
-5
55 BD 
-10
55 1C 
-8
E9 BD 
-11
1C 7A 
-1
55 55 
-1
1C 7A 
-6
1C 7A 
-7
1C E9 
-1
E9 E9 
2
BD 55 
10
1C 55 
-10
1C E9 
-1
55 E9 
6
7A 55 
-6
1C E9 
7
BD E9 
9
E9 55 
-6
E9 55 
3
1C 1C 
-10
1C BD 
-13
55 BD 
-7
BD 1C 
-10
7A 1C 
-7
BD 55 
9
7A 1C 
9
BD BD 
5
55 E9 
3
55 E9 
-5
BD 1C 
-13
E9 BD 
3
55 55 
-4
7A 7A 
-11
7A BD 
0
55 E9 
-7
E9 1C 
-4
55 BD 
-7
7A 1C 
6
7A 7A 
-8
E9 7A 
-4
BD 7A 
-5
7A 1C 
-8
BD 1C 
5
1C 7A 
-1
E9 7A 
6
7A 55 
-9
E9 55 
10
E9 55 
-11
BD BD 
1
55 BD 
9
7A 7A 
7
55 E9 
9
55 55 
-6
1C E9 
-13
7A E9 
-3
E9 55 
-5
55 1C 
-1
1C BD 
-4
BD E9 
-13
55 BD 
-13
BD 7A 
10
1C 7A 
4
7A BD 
-1
E9 55 
5
E9 7A 
7
55 E9 
9
1C 1C 
-3
E9 BD 
2
7A 1C 
3
E9 55 
-8
E9 1C 
-1
7A 1C 
9
7A 1C 
10
E9 55 
2
BD 7A 
4
7A 1C 
-1
7A 1C 
-10
55 1C 
-4
7A BD 
10
E9 1C 
8
1C E9 
9
BD 55 
-10
7A E9 
5
7A 55 
-9
BD 1C 
-7
7A E9 
-10
1C BD 
7
7A 7A 
-11
BD BD 
-9
7A E9 
-9
7A 1C 
5
55 BD 
-12
1C E9 
-12
7A 55 
8
E9 55 
-9
55 7A 
-5
7A BD 
-10
7A 55 
-10
E9 E9 
8
1C 55 
4
1C 7A 
-9
BD E9 
10
7A 55 
1
1C 7A 
6
1C E9 
4
1C 55 
-4
1C E9 
6
E9 E9 
4
BD E9 
-5
55 55 
-9
1C E9 
2
E9 55 
-13
BD 7A 
10
1C 1C 
10
55 E9 
-6
E9 1C 
-6
E9 1C 
1
7A 7A 
-7
1C 55 
-6
7A 1C 
-6
7A 55 
9
55 BD 
4
BD 7A 
-7
7A E9 
7
1C 55 
-9
7A 7A 
-10
7A BD 
-1
1C 7A 
6
BD E9 
-1
E9 7A 
-8
1C 55 
-3
BD 1C 
0
55 55 
-2
BD 1C 
1
BD E9 
7
E9 7A 
-5
55 E9 
5
7A 7A 
7
1C E9 
6
1C 7A 
-2
E9 E9 
-3
BD BD 
-7
7A 1C 
9
E9 55 
6
7A 55 
-12
E9 55 
4
E9 BD 
-12
E9 55 
4
55 55 
-9
55 7A 
6
55 E9 
5
1C 1C 
-9
BD E9 
4
7A BD 
-1
BD 7A 
-10
1C 1C 
10
1C 55 
8
BD E9 
4
7A 7A 
10
1C 1C 
7
BD 1C 
-5
1C BD 
-11
1C 55 
-6
7A BD 
-9
E9 7A 
0
E9 BD 
0
BD 55 
-11
E9 1C 
10
1C 1C 
-4
55 E9 
-11
7A 7A 
0
1C E9 
4
7A 1C 
-11
E9 55 
-7
E9 55 
-10
BD E9 
4
BD E9 
-5
BD E9 
-10
BD E9 
3
7A 1C 
-9
55 7A 
10
BD E9 
-4
E9 E9 
-12
7A E9 
0
7A 55 
-12
55 E9 
-12
BD E9 
-6
7A BD 
-4
7A E9 
-10
1C E9 
-7
E9 7A 
10
E9 55 
-3
BD 7A 
-8
E9 55 
-5
55 55 
-6
1C 1C 
-9
E9 E9 
-7
BD BD 
-4
BD 55 
4
E9 7A 
6
1C 55 
3
BD 55 
1
1C 55 
3
7A 55 
3
55 BD 
5
7A E9 
0
1C 55 
-7
1C E9 
-7
7A 1C 
-1
55 BD 
-9
E9 E9 
-11
1C E9 
-10
E9 BD 
-1
E9 7A 
-13
1C 1C 
1
E9 E9 
-13
7A 55 
-6
E9 55 
-13
1C 55 
0
7A 55 
8
1C 7A 
-4
E9 7A 
5
1C 55 
10
E9 BD 
-11
7A E9 
-9
1C 7A 
2
BD 7A 
3
BD 55 
8
1C 1C 
-5
55 BD 
-1
E9 E9 
5
55 55 
8
E9 BD 
-12
E9 7A 
0
1C 1C 
-13
1C 7A 
-1
1C BD 
-6
1C 1C 
-1
7A E9 
-5
E9 E9 
9
7A E9 
-3
1C 55 
-9
7A 1C 
-2
7A E9 
-1
1C 7A 
-1
7A E9 
-9
E9 E9 
5
1C 7A 
1
E9 BD 
-12
E9 55 
0
55 7A 
2
7A BD 
-1
1C E9 
6
55 BD 
1
7A 7A 
-2
1C E9 
8
7A 1C 
-3
55 1C 
8
7A 1C 
0
E9 55 
-1
55 E9 
6
55 E9 
-9
7A E9 
6
BD BD 
-2
55 E9 
-5
1C 7A 
-12
BD E9 
-5
55 7A 
2
1C 55 
4
55 1C 
7
1C 55 
4
7A 1C 
9
E9 E9 
2
7A BD 
9
BD E9 
1
7A BD 
6
1C 55 
-2
55 E9 
-5
55 BD 
-4
1C 7A 
7
7A E9 
-7
BD BD 
7
BD BD 
6
BD 7A 
6
BD 55 
-7
1C 1C 
-6
1C E9 
10
7A E9 
3
7A 1C 
-7
E9 E9 
4
55 1C 
-11
1C BD 
-8
1C BD 
-4
E9 7A 
2
55 55 
-10
7A BD 
3
1C E9 
2
BD 7A 
-6
55 55 
-3
BD E9 
9
E9 1C 
0
E9 BD 
1
7A 7A 
-12
E9 E9 
9
E9 7A 
8
55 55 
3
E9 7A 
-5
55 1C 
-12
1C BD 
0
BD 55 
-10
55 BD 
-5
1C 55 
-3
1C E9 
-13
55 55 
-11
7A BD 
-8
1C 7A 
-3
7A 55 
-12
BD E9 
0
1C E9 
0
BD E9 
-7
BD 1C 
10
7A 7A 
6
55 55 
1
7A E9 
0
1C 1C 
-5
BD E9 
1
E9 BD 
-4
55 55 
-2
7A E9 
-7
1C 1C 
-11
BD 1C 
-10
BD 7A 
8
E9 1C 
-8
7A 55 
-10
BD 1C 
-12
55 55 
-11
55 E9 
0
55 E9 
-3
E9 E9 
-7
55 1C 
-10
1C E9 
5
E9 E9 
1
E9 55 
-8
BD 7A 
8
7A BD 
3
7A E9 
-2
55 7A 
3
BD BD 
-10
BD 7A 
-12
1C BD 
7
BD BD 
1
55 7A 
6
BD BD 
-10
BD 55 
3
1C BD 
4
BD 1C 
-11
E9 1C 
2
E9 1C 
-10
55 E9 
6
7A BD 
-9
E9 BD 
7
55 E9 
-3
7A 55 
7
BD 55 
3
BD BD 
4
BD 55 
2
BD 7A 
-11
E9 E9 
6
1C E9 
-13
7A E9 
2
1C 7A 
7
55 BD 
4
55 7A 
-6
BD 7A 
3
BD BD 
-13
BD 1C 
-1
1C 7A 
-2
E9 E9 
-8
E9 7A 
-1
1C 1C 
-9
7A E9 
8
7A 55 
-1
55 1C 
1
1C BD 
-13
BD BD 
-6
7A BD 
1
1C BD 
-4
1C 55 
8
BD 7A 
-13
7A BD 
-1
E9 7A 
4
BD 55 
4
1C BD 
-1
BD 7A 
-2
1C 7A 
4
55 BD 
7
7A 55 
-10
55 E9 
9
1C 1C 
-13
1C 55 
-3
55 1C 
-9
1C E9 
-6
BD BD 
4
7A BD 
4
1C E9 
5
BD BD 
-13
E9 BD 
-9
1C E9 
0
BD 7A 
5
BD 1C 
2
E9 1C 
10
7A 1C 
-5
55 7A 
1
E9 1C 
-10
E9 7A 
-3
1C BD 
6
7A 7A 
-3
55 1C 
-6
55 7A 
-5
7A E9 
10
BD BD 
-13
7A 1C 
-8
E9 1C 
3
1C 55 
6
E9 E9 
-10
55 E9 
-1
1C 1C 
-11
E9 BD 
-2
BD E9 
-13
BD 55 
8
7A 55 
-8
E9 7A 
-1
BD 7A 
-7
E9 55 
-10
1C E9 
8
1C 7A 
-11
BD 7A 
1
1C 55 
-2
E9 BD 
-2
55 BD 
-13
BD 1C 
1
55 1C 
9
7A 7A 
-8
BD E9 
5
E9 BD 
-7
55 1C 
-7
7A E9 
-4
55 E9 
1
55 7A 
10
BD 7A 
-13
55 7A 
0
1C 7A 
-11
55 7A 
6
7A 1C 
5
7A BD 
-10
7A BD 
1
1C E9 
-5
7A 7A 
6
BD 1C 
-1
E9 7A 
9
E9 1C 
-12
7A 7A 
-7
BD 1C 
2
BD 1C 
-10
7A 55 
-12
55 BD 
5
1C BD 
10
55 7A 
-12
55 E9 
-8
1C E9 
7
1C E9 
-2
E9 1C 
5
1C 7A 
-6
7A 55 
-13
1C E9 
3
BD 7A 
-13
1C 7A 
8
55 7A 
-3
BD E9 
1
7A 7A 
2
1C E9 
-5
55 1C 
0
55 1C 
-4
55 7A 
3
E9 7A 
-6
1C 1C 
-3
E9 7A 
-2
1C E9 
-7
BD 1C 
3
1C 1C 
-9
55 55 
8
BD E9 
-12
E9 BD 
-4
E9 E9 
7
BD 1C 
-8
BD 55 
1
1C E9 
10
55 1C 
4
55 55 
3
7A 55 
3
1C 7A 
-5
E9 BD 
-4
1C 55 
-10
55 7A 
-2
E9 1C 
3
55 55 
-3
BD 1C 
-3
7A 7A 
-1
55 55 
-8
7A 7A 
4
7A 1C 
8
BD 1C 
-3
1C BD 
-13
7A 1C 
-4
55 1C 
-3
1C BD 
4
BD BD 
9
7A 7A 
-1
1C 1C 
-3
1C 55 
-5
BD E9 
5
BD 1C 
-11
BD 55 
4
55 7A 
7
1C 1C 
6
E9 1C 
-3
E9 1C 
-2
7A 7A 
-13
BD 55 
-5
BD 55 
-2
7A 7A 
3
E9 55 
-2
55 7A 
10
BD 55 
7
E9 BD 
7
E9 7A 
-3
1C E9 
3
E9 55 
6
1C BD 
-9
BD E9 
-6
BD E9 
-8
7A 1C 
-2
55 55 
-8
E9 1C 
-8
7A 1C 
-9
1C 7A 
4
E9 E9 
-5
7A 7A 
-8
7A E9 
7
BD 7A 
9
55 55 
-4
1C 7A 
2
BD 55 
-11
1C 55 
-6
BD 7A 
-11
BD 1C 
9
7A 1C 
6
BD 7A 
-10
55 1C 
-3
7A 7A 
-1
7A 55 
2
BD 1C 
-7
7A 7A 
1
1C E9 
10
BD E9 
-5
BD 7A 
8
BD E9 
-4
7A BD 
-3
7A 55 
7
1C BD 
-1
E9 1C 
1
E9 1C 
5
1C E9 
-10
1C E9 
-3
7A BD 
-2
BD 1C 
10
E9 E9 
-1
7A E9 
-8
55 1C 
-13
7A E9 
-7
E9 55 
-5
1C E9 
-9
BD E9 
-11
E9 E9 
1
55 1C 
-7
7A 1C 
1
55 55 
-2
BD BD 
-4
7A 55 
3
E9 7A 
0
E9 7A 
-5
E9 E9 
10
7A BD 
-4
7A 7A 
-9
55 1C 
-7
1C 1C 
-10
7A BD 
8

