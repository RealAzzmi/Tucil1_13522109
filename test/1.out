24
55 E9 1C BD 55 7A 
9, 1
9, 7
3, 7
3, 4
4, 4
4, 1

3016.86 ms